runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -247,7 +252,7 @@ protected ILogData read(final long address, long readStartTime) {
                 return runtime.getAddressSpaceView().read(address);
             }
             return runtime.getAddressSpaceView()
-                    .read(Collections.singleton(address), false)
+                    .read(Collections.singleton(address))
```

Comment:
Why is this behavior changed? It used to be NOT waitForWrite, but now it waits instead. Is that true?

Reply:
Leftover from the previous change. Good catch, fixed it.